doc/tutorials.rst

```diff
@@ -85,6 +85,11 @@ algorithms using PennyLane and near-term quantum hardware.
     :figure: ../examples/figures/qaoa_maxcut_partition.png
     :description: :ref:`qaoa_maxcut`
 
+.. customgalleryitem::
+    :tooltip: Barren plateaus in quantum neural networks
+    :figure: ../examples/figures/barren_plateaus/surface.png
+    :description: :ref:`barren_plateaus`
+
```

Comment:
While we will likely change this down the line (because the sidebar is getting a bit long), don't forget to add the tutorial to the TOC!